packages/utils/src/libp2p/index.ts

```diff
@@ -185,7 +184,7 @@ export async function dial(
   } catch (err) {
     logError(`Using new addresses after querying the DHT did not lead to a connection. Cannot connect. ${err.message}`)
     clearTimeout(timeout)
-    return { status: 'E_DIAL', error: err, dht: true }
+    return { status: 'E_DIAL', error: err.message }
```

Comment:
This means that we cannot differentiate between errors before or after the DHT lookup - perhaps append a string here.

Reply:
Ahh, now I get the reason for the DHT property!

Reply:
Going to rename it slightly

Reply:
done